core/src/main/java/hudson/model/UpdateSite.java

```diff
@@ -1043,6 +1043,18 @@ public PluginWrapper getInstalled() {
             return pm.getPlugin(name);
         }
 
+        @Exported
+        public boolean isCompatible() {
+            return isCompatible(new PluginManager.MetadataCache());
+        }
+
+        @Exported
```

Comment:
Cannot be exported due to argument, so should be throwing an exception in the remote API (or at least log a warning that this is un-exportable)

Reply:
I'm not sure how the @Exported annotation is being used. I included it in both cases as I saw it in the original "isCompatibleWithInstalledVersion", but now that I'm looking to it in more detail, I assume it is only used as a way to publish the method as an API operation, right?

If that is the case, it seems that for this method overload the annotation is not necessary at all.

Reply:
Removed and tested locally afterwards, working fine.